sequence_task_3/student/src/impl.cpp

```diff
@@ -1,2 +1,19 @@
 #include "impl.h"
+#include <algorithm>
+#include <string>
+#include <numeric>
+#include <iostream>
 
+std::string reverseWords(const std::string& input) {
+    std::deque<std::string> strDeq;
+    std::string world;
+    std::stringstream myString(input);
+
+    while (myString >> world) {
+        strDeq.push_front(world);
+        strDeq.push_front(" ");
+    }
+
+    std::string result = std::accumulate(strDeq.begin(), strDeq.end(), result);
```

Comment:
std::accumulate should never be used with strings!
Reason: https://stackoverflow.com/questions/15347123/how-to-construct-a-stdstring-from-a-stdvectorstring

Prefer stream iterators!